src/iop/colorin.c

```diff
@@ -1938,6 +1941,27 @@ void reload_defaults(dt_iop_module_t *module)
       }
     }
 #endif
+#ifdef HAVE_LIBHEIF
```

Comment:
We need a runtime detection if libheif has been compiled with heic support here!